validation-phase-workflow.cwl

```diff
@@ -21,65 +24,102 @@ inputs:
   synapseConfig:
     label: filepath to .synapseConfig file
     type: File
+  organizers:
+    label: User or team ID for challenge organizers
+    type: string
+    default: "3487813"
 
 outputs: {}
 
 steps:
+  organizers_log_access:
+    doc: >
+      Give challenge organizers `download` permissions to the submission logs
+    run: |-
+      https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v4.1/cwl/set_permissions.cwl
+    in:
+      - id: entityid
+        source: "#submitterUploadSynId"
+      - id: principalid
+        source: "#organizers"
+      - id: permissions
+        valueFrom: "download"
+      - id: synapse_config
+        source: "#synapseConfig"
+    out: []
 
   set_submitter_folder_permissions:
-    run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v3.1/cwl/set_permissions.cwl
+    run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v4.1/cwl/set_permissions.cwl
     in:
       - id: entityid
         source: "#submitterUploadSynId"
       # TODO: replace `valueFrom` with the admin user ID or admin team ID
       - id: principalid
-        valueFrom: "3487816"
+        valueFrom: "#organizers" 
       - id: permissions
         valueFrom: "download"
       - id: synapse_config
         source: "#synapseConfig"
     out: []
-
-  download_submission:
-    run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v3.1/cwl/get_submission.cwl
+    
+    download_submission:
+    doc: Download submission
+    run: |-
+      https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v4.1/cwl/get_submission.cwl
     in:
       - id: submissionid
         source: "#submissionId"
       - id: synapse_config
         source: "#synapseConfig"
     out:
       - id: filepath
-      - id: docker_repository
-      - id: docker_digest
       - id: entity_id
       - id: entity_type
+      - id: evaluation_id
       - id: results
       
+  create_config_file:
+    run: /bin/bash
```

Comment:
I think this is missing a shell command?

Reply:
This is a placeholder for when I get the workflow solidified from the organizers but I commented it out for now.

Reply:
Ohh, makes sense.  Yes, commenting it out for now is a good idea. Thanks!